google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractResultSet.java

```diff
@@ -655,6 +670,26 @@ public boolean isNull(int columnIndex) {
       return rowData.get(columnIndex) == null;
     }
 
+    @Override
+    public <T extends AbstractMessage> T getProtoMessageInternal(int columnIndex, T m) {
+      Preconditions.checkNotNull(
+          m, "Proto Message shouldn't be null, default instance from generated class can be used.");
```

Comment:
just curious, why should `Proto Message shouldn't be null`? 

If it is within the scope of this method only, can reword comment better.